CODEOWNERS

```diff
@@ -1,2 +1,4 @@
 *               @ktsakalozos
```

Comment:
Now that the addons are removed from the source tree we can remove this file. The entire team is responsible for all the code.

Reply:
@neoaggelos please remove the CODEOWNERS file.